### PR TITLE
Clean up docs (grammar and code style)

### PR DESCRIPTION
**Pull request by thomastuts**
This PR fixes the following in the docs:

- grammar mistakes (there might be more, I haven't read through all of the docs yet)
- code style (add missing semicolons, remove shorthand if-statements)

I noticed the source code also uses a mix of regular and shorthand if-statements. The doc fixes are in a separate commit, so it's easy to pull those out if you'd rather keep the shorthand version.

### CODE REVIEW

File: doc/01 Route Configuration/Plain Route.md [hunk 1]
```diff
@@ -48,10 +48,12 @@ var myRoute = {
   path: 'picture/:id',
   getChildRoutes (state, cb) {
     // state gets passed to `getChildRoutes`
-    if (state && state.fromDashboard)
+    if (state && state.fromDashboard) {
       cb(null, [dashboardPictureRoute])
-    else
+    }
+    else {
```
Comment (cesarandreu) [inline doc/01 Route Configuration/Plain Route.md]: I've never seen anyone add a newline before the else

Comment (thomastuts) [inline doc/01 Route Configuration/Plain Route.md]: Yeah, that's mostly a personal preference, honestly. I feel like having `else` on the same line as the closing curly brace for `if` kind of makes `else` feel like a 'second-class citizen', even though its logic might be just as (or even more) important.

I realize I'm most likely in the minority regarding this, so if that's something that should be changed, I'd be happy to.

Comment (mjackson) [inline doc/01 Route Configuration/Plain Route.md]: Can you please stick with existing code style? Thanks!

**Comment by knowbody** [comment]
@thomastuts I'm happy to merge that can you resolve the issues, keep the project style as the guys mentioned in comments and then rebase? thanks

**Comment by ryanflorence** [comment]
thanks for the work on this, we've been moving pretty fast and the docs are all different now, sorry we didn't merge it :(

**Pull request closed without merging**